src/SelectMulti.vue

```diff
@@ -272,12 +272,14 @@
 				}
 			},
 			onInputBlur() {
-				if (this.ignoreBlur) {
-					this.ignoreBlur = false
-					return
-				}
-
-				this.updateMenuState(false, false)
+				setTimeout(() => {
```

Comment:
Would be great to have a comment here why we need setTimeout
```suggestion
				// To force onInputBlur run after updateMenuState we use setTimeout
				setTimeout(() => {
```